### DISCUSSION TITLE
How to troubleshoot renovate not running hourly?

### DISCUSSION THREAD

**Discussion by richardsimko** [renovatebot/renovate#8389] [Request Help] [answered]
I've enabled renovate across some 20 or so repositories and for all apart from 3 it seems to be working flawlessly. However for these 3 it doesn't run on its hourly schedule (It doesn't run at all, having been enabled for close to 48 hours in some of these and the only runs are manual).

I've tried removing its access to these repos and re-adding it but this only triggered a single run. For some of the repos it triggers on commits to master, on others not.

Is there something I can look for in the log to troubleshoot this? All of the repositories have the same shared config, so it shouldn't be down to a broken config.

**Comment by viceice** [comment, marked as answer]
check the logs on https://app.renovatebot.com/dashboard

**Reply by viceice** [reply]
If renovate hosted app don't find any deps it will reduce schedule to a daily run.

**Reply by richardsimko** [reply]
Yeah I've been looking at those but what should I look for? It has found deps since it runs properly on a manual run, there are also some outstanding PRs which it should automerge according to the config.

**Reply by richardsimko** [reply]
The log results all look like this with all 3 runs being manual runs. No schedule is configured so it should be allowed to run whenever.

![image](https://user-images.githubusercontent.com/1245031/105507202-33aca100-5ccb-11eb-9a13-c9b14fe999f3.png)

**Reply by viceice** [reply]
@rarkins Can you check the backend?

**Reply by rarkins** [reply]
Onboarding repositories are only scheduled once per day, but should still run due to webhooks e.g. if you edit the PR files

**Reply by richardsimko** [reply]
But these repositories should have been onboarded since several days. This is for normal dependency checks, not for the onboarding PR. 

The logs also say:

```
DEBUG: Repo is onboarded
```

**Reply by richardsimko** [reply]
I enabled Renovate in these repos at the same time as several others where it works (Roughly, give or take a few hours).

**Reply by rarkins** [reply]
OK. we have a weird bug where that status sometimes takes a while to get switched over. I have updated it manually for 3 of your repos which were in onboarding state

**Reply by richardsimko** [reply]
Thanks, that seems to have done the trick, however I have another repo which seems to be stuck in the same state. Could you enable it for that one as well? It also hasn't run for 14 hours, which I believe was a manual run

![image](https://user-images.githubusercontent.com/1245031/105579796-fd355b80-5d88-11eb-8e28-cc8120d307cd.png)

**Reply by richardsimko** [reply]
It seems to be working now! Thanks for the help! :)
